diff -Nuarp mesa-25.0.5.orig/src/vulkan/util/vk_extensions.py mesa-25.0.5/src/vulkan/util/vk_extensions.py
--- mesa-25.0.5.orig/src/vulkan/util/vk_extensions.py	2025-05-01 01:18:00.000000000 +0800
+++ mesa-25.0.5/src/vulkan/util/vk_extensions.py	2025-05-03 14:12:06.383127710 +0800
@@ -50,6 +50,7 @@ class Extension:
         return ext
 
     def c_android_condition(self):
+        return 'true'
         # if it's an EXT or vendor extension, it's allowed
         if not self.name.startswith(ANDROID_EXTENSION_WHITELIST_PREFIXES):
             return 'true'
